components/screens/menu/Version.tsx

```diff
@@ -0,0 +1,15 @@
+import * as Application from 'expo-application';
+import * as Updates from 'expo-updates';
+import {getUpdateTimeAsVersionString} from 'hooks/useEASUpdateStatus';
+import {Platform} from 'react-native';
+
+export const getVersionInfoFull = (mixpanelUserId: string, updateGroupId: string): string => {
```

Comment:
could either mixpanel or update group Id ever be null and do we care if they are?

Reply:
I think at least one of them can be, which is why you're getting the error you're getting above:

>  Argument of type 'string | undefined' is not assignable to parameter of type 'string'.

Reply:
so whats the best way to handle this? would we just pass an empty string to the version function if it is null for some reason or have the version function take untyped variable and handle it possibly being null in there? I played around with this a bit, but couldn't find a solution that looked right to me 🤷